wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -83,12 +83,14 @@ import           Universum
 
 import           Data.Semigroup (Semigroup)
 
+import           Cardano.Wallet.API.V1.Swagger.Example (Example, example)
 import           Control.Lens (At, Index, IxValue, at, ix, makePrisms, to, (?~))
 import           Data.Aeson
 import           Data.Aeson.TH as A
 import           Data.Aeson.Types (toJSONKeyText, typeMismatch)
 import qualified Data.Char as C
-import           Data.Swagger as S
+import           Data.Swagger as S hiding (Example, example)
+import qualified Data.Swagger as SE (example)
```

Comment:
We can keep `S` instead of `SE` here. They refer to the same module anyway :)

Reply:
Actually, there was a reason. This way I could simply copy-paste the `Example`-instances and have `example` refer to `Swagger.Example.example`, and accessing `Data.Swagger.example` as `SE.example`.

So we *can*, but have to make 50+ references to `Example` and `example` qualified. Feels a little bit wrong, but I guess explicitly is good?

Reply:
Sorry for the delay, but what I meant was simply to have:

```hs
import           Data.Swagger hiding (Example, example)
import qualified Data.Swagger as S
```

Even though, I'd rather list everything we use from `Data.Swagger` if it's realistic.

Reply:
Seems borderline unrealistic (2k LoC and lots of things from `Data.Swagger`). Should I spend time on it? Could perhaps get [importify](https://github.com/serokell/importify) working.

Reply:
("unrealistic"  as in I got to like 10-20 symbols and then gave up, and not sure if there are 25 in total or 100)